stor/tests/test_dx.py

```diff
@@ -813,6 +893,11 @@ def test_project_does_not_exist(self):
 
 
 class TestGlob(DXTestCase):
+    def test_empty_path(self):
+        dx_p = DXPath('dx://')
+        with pytest.raises(ValueError, match='not supported'):
+            dx_p.glob('*')
```

Comment:
wouldn't the glob apply to project names generated?

Reply:
Glob is implemented internally as walkfiles. Meaning it only finds the files within the path. So ideally we should be finding all the files a user has access to across projects and returning the ones that match the glob pattern. Implementing it this way may introduce some confusion.